app/views/api_docs/reference/v0.1/_reference.md

```diff
@@ -705,24 +747,29 @@ Trainee details
     <pre class="json-code-sample">
     {
       "data": {
+        "provider_trainee_id": "12345678",
         "first_names": "John",
-        "middle_names": "James",
         "last_name": "Doe",
         "date_of_birth": "1990-01-01",
-        "sex": "male",
+        "sex": "11",
         "email": "john.doe@example.com",
-        "trn": "123456",
         "training_route": "11",
-        "itt_start_date": "2022-09-01",
-        "itt_end_date": "2023-07-01",
-        "diversity_disclosure": "diversity_disclosed",
-        "ethnic_group": "white_ethnic_group",
-        "ethnic_background": "Background 1",
+        "itt_start_date": "2023-09-01",
+        "itt_end_date": "2024-07-01",
         "course_subject_one": "100425",
         "study_mode": "01",
-        "application_choice_id": "123",
+        "nationality": "GB",
+        "ethnicity": "120",
+        "disability1": "58",
+        "itt_aim": "201",
+        "itt_qualification_aim": "004",
+        "course_year": "2",
+        "course_age_range": "13918",
+        "fund_code": "7",
+        "funding_method": "4",
+        "hesa_id": "1210007145123456",
         "placements_attributes": [{ "urn": "123456", "name": "Placement" }],
-        "degrees_attributes": [{ "country": "UK", "grade": "First", "subject": "Computer Science", "institution": "University of Test", "graduation_year": "2012", "locale_code": "uk" }]
+        "degrees_attributes": [{ "grade": "02", "subject": "100425", "institution": "0116", "uk_degree": "083", "graduation_year": "2012-06-31"}]
```

Comment:
Yes, this is a bit of a strange one - we only care about the year, but the relevant HESA data is a full date so we're accepting that and only storing the year. Probably something we'll come back to because it is not very satisfactory.

Date and formatting updated in https://github.com/DFE-Digital/register-trainee-teachers/pull/4219/commits/d9053ba5d2fdfa5011383b1aa6cc21c895074f75